cmd/kube-apiserver/app/server.go

```diff
@@ -626,7 +626,7 @@ func Complete(s *options.ServerRunOptions) (completedServerRunOptions, error) {
 				return options, fmt.Errorf("error finding host name: %v", err)
 			}
 		}
-		klog.Infof("external host was not specified, using %v", s.GenericServerRunOptions.ExternalHost)
+		klog.InfoS("external host was not specified, using %v", s.GenericServerRunOptions.ExternalHost)
```

Comment:
ditto

Reply:
A capital letter  is suggested, and the format is error. Pls start after referring to  https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/1602-structured-logging and https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md